lib/web/apiserver.go

```diff
@@ -757,6 +763,17 @@ func (h *Handler) find(w http.ResponseWriter, r *http.Request, p httprouter.Para
 	}, nil
 }
 
+func (h *Handler) getClusterCA(w http.ResponseWriter, r *http.Request, p httprouter.Params) (interface{}, error) {
+	localCAResponse, err := h.cfg.ProxyClient.GetClusterCACert()
```

Comment:
Do you know if this includes all CAs (user, host, jwt) or only host CA?

Reply:
This is only the host CA TLS certs